option.go

```diff
@@ -278,10 +278,14 @@ func WithHTTPGet() ClientOption {
 //
 // By default, Connect-protocol clients with GET requests enabled may send a
 // URL of any size.
-func WithHTTPGetMaxURLSize(bytes int, fallback bool) ClientOption {
+func withHTTPGetMaxURLSize(bytes int, fallback bool) ClientOption {
 	return &getURLMaxBytes{Max: bytes, Fallback: fallback}
 }
 
+// TODO(jchadwick-buf): Remove this once we've determined whether this option
+// should be exported or removed.
+var _ = withHTTPGetMaxURLSize(0, false)
```

Comment:
This is ugly, but otherwise the linter (correctly) determines that the option is unused.